aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>This last part adds confusion... what kind of model? The first model is an audio model - what's the second one?</p><p> </p><p>Might be better just with this:</p><p>"Improve an audio model by training with supplemental synthetic features"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I am actually using two models. One is a pre-trained model from AWS marketplace and the other is a downstream audio classification model. Does this sound better - "Supplement your data with synthetic features from a pre-trained model to train a better downstream audio classification model" ? </p>